plugin-server/src/main/ingestion-queues/analytics-events-ingestion-overflow-consumer.ts

```diff
@@ -0,0 +1,51 @@
+import Piscina from '@posthog/piscina'
+import * as schedule from 'node-schedule'
+
+import { KAFKA_EVENTS_PLUGIN_INGESTION_OVERFLOW, prefix as KAFKA_PREFIX } from '../../config/kafka-topics'
+import { Hub } from '../../types'
+import { status } from '../../utils/status'
+import { eachBatchIngestion } from './batch-processing/each-batch-ingestion'
+import { IngestionConsumer } from './kafka-queue'
+
+export const startAnalyticsEventsIngestionOverflowConsumer = async ({
+    hub, // TODO: remove needing to pass in the whole hub and be more selective on dependency injection.
+    piscina,
+}: {
+    hub: Hub
+    piscina: Piscina
+}) => {
+    /*
+        Consumes analytics events from the Kafka topic `events_plugin_ingestion_overflow`
+        and processes them for ingestion into ClickHouse.
+
+        This is the overflow or "slow-lane" processing queue as it contains only events that
+        have exceed capacity.
+
+        At the moment this is just a wrapper around `IngestionConsumer`. We may
+        want to further remove that abstraction in the future.
+    */
+    status.info('🔁', 'Starting analytics events overflow consumer')
+
+    // NOTE: we are explicitly not maintaining backwards compatibility with
+    // previous functionality regards to consumer group id usage prior to the
+    // introduction of this file. Previouslty, when ingestion and export
+    // workloads ran on the same process they would share the same consumer
+    // group id. In these cases, updating to this version will result in the
+    // re-exporting of events still in Kafka `clickhouse_events_json` topic.
+
+    const queue = new IngestionConsumer(
+        hub,
+        piscina,
+        KAFKA_EVENTS_PLUGIN_INGESTION_OVERFLOW,
+        `${KAFKA_PREFIX}clickhouse-ingestion-overflow`,
```

Comment:
God I'd love to change these groupId names, but for consistency this is good.

Reply:
maybe it's better if we change it, because we were thinking of making the normal ingestion topic have more partitions and if we do that we can rename that one, and not need to come back to this one. On that note - how many partitions do we have for this one?

Reply:
128